spec/kitchen/driver/openstack_spec.rb

```diff
@@ -1144,6 +1170,16 @@
     end
   end
 
+  describe '#wait_for_server' do
+    let(:state) { { hostname: 'hostname' } }
+
+    context 'standard WinRM sleep check' do
+      it 'calls the normal Kitchen WinRM wait' do
+        expect_any_instance_of(described_class).not_to receive(:sleep)
```

Comment:
This feels like a strange test to me, and that's without looking at the code.  There's no other context defined where sleep SHOULD be called, so perhaps instead we should check to make sure the intended method is called vs. a method NOT being called.